primitives/src/policy.rs

```diff
@@ -502,7 +502,8 @@ impl Policy {
         let exponent = -Policy::SUPPLY_DECAY * t;
 
         let supply = genesis_supply
-            + (Self::INITIAL_SUPPLY_VELOCITY / Self::SUPPLY_DECAY * (1.0 - exp(exponent))) as u64;
+            + (Self::INITIAL_SUPPLY_VELOCITY / Self::SUPPLY_DECAY
+                * (1.0 - f64::powf(2.0, exponent))) as u64;
```

Comment:
We can't use the `f64::powf` function here, since this is consensus-relevant and https://doc.rust-lang.org/std/primitive.f64.html#method.powf says:
> The precision of this function varies by platform and Rust version.

Reply:
Ok, I added `pow` to our math utils and changed the function to use it.